contrib/containerized-pbench/pbench_demo

```diff
@@ -20,16 +20,18 @@ export PB_AGENT_IMAGE_NAME=quay.io/pbench/pbench-agent-all-fedora-36:main
 
 mkdir -p ${FIOTEST}
 
+# Before we run the demo, we need a Pbench authentication token. The token
+# location need to be supplied to this script as an argument.
+token_location=$1
+token=$(< ${token_location})
 #+
 # Run the demo!
 #-
-pbench pbench-generate-token --output=/var/lib/pbench-agent/.token
 pbench pbench-register-tool-set light
 pbench pbench-list-tools
 pbench pbench-user-benchmark --config example-workload -- \
     fio --directory=/fiotest --name fio_test_file --direct=1 --rw=randread \
         --bs=16k --size=100M --numjobs=8 --time_based --runtime=5s \
         --group_reporting --norandommap
-# Note that the token file location below is evaluated -outside- the container,
-# unlike in the pbench-generate-token command above.
-pbench pbench-results-move --token=$(< /var/tmp/${USER}/pbench-agent/run/.token)
+# Note that the token file location below is evaluated -outside- the container.
+pbench pbench-results-move --token=token
```

Comment:
The comment is incorrect, since you dereferenced the `token_location` above. However, the code is also wrong: I think you meant `--token=${token}`.

I also kinda wonder whether we want to require that the token be specified in a file, vs allowing `${1}` to be the actual token, e.g., `pbench_demo $(< /file)` if you really *want* or simply `pbench_demo <token>`. Or we could be fancy and allow `pbench_demo @<file>` as well as `pbench_demo <token>`...

Reply:
Under the assumption that this PR is _not_ going to modify `pbench-results-move` (and given that  the tool expects the value specified by `--token` to be the token string), I would just have the `pbench_demo` script require that the token be available in an environment variable which it references here and let the user worry about how to get set that up.

(Yeah, it would be cool if `pbench-results-move` supported the `@<file>` syntax, but that's for a different PR.)